src/api/middleware/validators/company.js

```diff
@@ -30,7 +30,7 @@ export const list = useExpressValidators([
     query("offset", ValidationReasons.DEFAULT)
         .optional()
         .isInt({ min: 0 })
-        .withMessage(ValidationReasons.MIN(0)),
+        .withMessage(ValidationReasons.MIN(ValidationReasons.MIN)),
```

Comment:
This doesn't seem right

Reply:
I think 0 is fine in this situation, because it's an offset of a query

Reply:
Yeah this one doesn't make sense, I was testing in the beginning and forgot to change this one back. Will make the rest of the adjustments today